admin/sales_analytics/metrics.py

```diff
@@ -0,0 +1,113 @@
+from admin.base.settings import ENTRY_POINTS
+from admin.sales_analytics.utils import get_entry_point, get_sorted_index
+from datetime import datetime, timedelta
+from framework.mongo import database as db
+
+
+def get_user_count(db=db, entry_points=ENTRY_POINTS):
+    """
+    Get the number of users created from each entry point: osf, osf4m, prereg, and institution.
+    """
+    count_list = []
+    percent_list = []
+    tags = ['osf4m', 'prereg', 'institution', 'osf']
```

Comment:
It seems like these tags mirror the entry points with the addition of 'osf.' This means that every time you update `ENTRY_POINTS` you will have to update tags. And, if I am not mistaken, keep them in the same order, or risk mislabeling data. I would suggest that you use `ENTRY_POINTS` as a basis list and then add 'osf' after you are done with the for loop.